### DISCUSSION TITLE
Downgrading chromebook issues

### DISCUSSION THREAD

**Discussion by pl4udible** [3kh0/ext-remover#143] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Yo, if possible can you please unpatch it because my organisation does not allow me to open the files. 

BTW, if anyone wants to know how to inspect on a school chromebook be my guest and check my post

**Comment by ghost** [comment, marked as answer]
what files? also, use this bookmarklet: `javascript:(function () {     var script =  document.createElement('script');    script.src="//cdn.jsdelivr.net/npm/eruda";     document.body.appendChild(script);    script.onload = function () {         eruda.init()     } })();` to inspect a page

**Reply by pl4udible** [reply]
The zipped file when uwant to downgrade it

**Comment by 3kh0** [comment]
See https://chrome100.dev for downgrading

**Reply by pl4udible** [reply]
My school chromebooks dont allow me to open it but download it

**Reply by ghost** [reply]
then you shouldn't really be downgrading. you seem to not know what you're doing
